sources/packages/web/src/components/layouts/Institution/sidebar/HomeSideBar.vue

```diff
@@ -22,15 +24,23 @@ export default {
   setup() {
     const store = useStore();
     const router = useRouter();
-
     const userLocationList = computed(() => store.state.institution.locationState);
+    const isAdmin = computed(() => store.state.institution.userState?.isAdmin);
```

Comment:
The code looks good, but as a recommendation for now or the future, I believe that we can use the composition API to aggregate these common-auth properties under the same hook, like below. In this way we would avoid doing the below line over and over again.
```
const isAdmin = computed(() => store.state.institution.userState?.isAdmin);
```

I believe that we could have something like:

```
const auth = useInstitutionAuth();
```
where auth would be the computed properties like isAdmin and userAuth(authorizations);
Please consider it as a recommendation. We can handle this in the future if appropriate.